packages/jsii/lib/assembler.ts

```diff
@@ -335,46 +329,29 @@ export class Assembler implements Emitter {
       // we expect it to appear as a peer dependency instead of a normal dependency.
       if (assembly) {
         if (!(assembly.name in this.projectInfo.peerDependencies)) {
-          this._diagnostic(
-            referencingNode,
-            ts.DiagnosticCategory.Warning,
-            `The type '${ref}' is exposed in the public API of this module. ` +
-              `Therefore, the module '${assembly.name}' must also be defined under "peerDependencies". ` +
-              'This will be auto-corrected unless --no-fix-peer-dependencies was specified.',
+          this._diagnostics.push(
+            JsiiDiagnostic.Code.JSII_0005_MISSING_PEER_DEPENDENCY.create(
```

Comment:
`JsiiDiagnostic.Code` feels rather verbose. Why not one top-level class? Ultimately we only care about the predefined `JSII_0005_MISSING_PEER_DEPENDENCY` (et al) constants, no? Why not make them easier to access?

Reply:
You're probably right. I've been self-debating this myself, too.